src/main/java/core/basesyntax/supliers/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.supliers;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private static final int FIGURE_INDEX = 4;
+    private static final int CONSTANT_RADIUS = 10;
+    private final Random random = new Random();
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE.toString(), CONSTANT_RADIUS);
```

Comment:
```suggestion
        return new Circle(Color.WHITE.name(), CONSTANT_RADIUS);
```
And also make the color a constant, too